packages/package-template/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "package-template",
+  "module": "index.ts",
```

Comment:
This is the "magic" to let the example-app consume this package, bypasses any build within the package and lets the consumer build the TS (assuming it's configured to do so)

Reply:
Thank's a lot